src/js/common/components/AlertBox.jsx

```diff
@@ -33,7 +33,7 @@ class AlertBox extends React.Component {
 
   render() {
     if (!this.props.isVisible) {
-      return <div aria-live="assertive"/>;
+      return null;
```

Comment:
So the reasoning behind this is that there will always be a DOM element rendering, and if the AlertBox has content, it will re-render in the same place but with children. 

Because screenreaders are inconsistent in the way they handle content randomly being inserted into the DOM, it is best to always have a wrapper div with `aria-live` so that the reader knows that the content inside may change (and will alert the user accordingly).